generators/entity-server/templates/src/main/java/package/repository/EntityRepository.java.ejs

```diff
@@ -112,6 +113,14 @@ public interface <%= entityClass %>Repository extends <% if (databaseType === 's
 
     @Query("<%- (databaseType === 'mongodb')?"{'id': ?0}":"#{#n1ql.selectEntity} USE KEYS $1 WHERE #{#n1ql.filter}"%>")
     Optional<<%= asEntity(entityClass) %>> findOneWithEagerRelationships(<%= primaryKeyType %> id);
+    <%_
+        } else if (databaseType === 'neo4j') { _%>
+
+    @Query("MATCH (n:<%= asEntity(entityClass) %>)<-[]-(m) RETURN n,m")
+    Page<<%= asEntity(entityClass) %>> findAllWithEagerRelationships(Pageable pageable);
+
+    @Query("MATCH (e:<%= asEntity(entityClass) %> {id: ?0}) RETURN e")
```

Comment:
```suggestion
    @Query("MATCH (e:<%= asEntity(entityClass) %> {id: $id}) RETURN e")
```

According to https://neo4j.github.io/sdn-rx/current/#howdoispecifyparametersincustomqueries it should be `$id`

Reply:
thanks, done !